lib/resolvers/project.js

```diff
@@ -324,6 +324,13 @@ module.exports = ({ models }) => async ({ action, data, id, meta = {} }, transac
     return newProject;
   }
 
+  if (action === 'transfer-draft') {
+    return Promise.resolve()
+      .then(() => Project.query(transaction).patchAndFetchById(id, {
+        establishmentId: data.establishmentId
+      }));
+  }
```

Comment:
I assume we have some validation earlier on in this journey to prevent you transferring to an establishment you aren't affiliated with, and to prevent transferring to the same est?

Reply:
We do both of those in the UI, but I'll add an API level check as well.

Reply:
https://github.com/UKHomeOffice/asl-public-api/pull/264/files#diff-53d8536774fb33e1aec6b7d8fce33b5eR221-R244

Reply:
👍